metadata.json

```diff
@@ -5,7 +5,7 @@
   "summary": "This module provides a native type and provider to manage keys and values in the Windows Registry",
```

Comment:
Here is one where the author is still puppetlabs. You can choose to change or we can leave as is.